### PR TITLE
Update docs to reflect new change to setting release and dist in React Native

### PR DESCRIPTION
**Pull request by jennmueng**
The `Sentry.setRelease` and `Sentry.setDist` have been deprecated and setting these values have been moved to just passing them to `Sentry.init`

https://github.com/getsentry/sentry-react-native/pull/1009

### CODE REVIEW

File: src/docs/platforms/react-native/codepush.mdx [hunk 1]
```diff
@@ -5,19 +5,40 @@ toc: true
 title: Using Sentry with CodePush
 tags: []
 ---
-If you want to use Sentry together with CodePush you have to send us the CodePush version:
+If you want to use Sentry together with CodePush you will have to pass `release` and `dist` to `Sentry.init`. Our suggestion is to store them in an `app.json`, or you can just use the `package.json`. These values need to be unique to each version of your codebase and match the version on the sourcemaps exactly or they might not be symbolicated.
```
Comment (MimiDumpling) [inline src/docs/platforms/react-native/codepush.mdx:8]:
```suggestion
If you want to use Sentry together with CodePush you will have to pass `release` and `dist` to `Sentry.init`. Our suggestion is to store them in an `app.json`, or you can just use the `package.json`. These values need to be unique to each version of your codebase and match the version on the source maps exactly, or they might not be symbolicated.
```

File: src/docs/platforms/react-native/codepush.mdx [hunk 2]
```diff
@@ -5,19 +5,40 @@ toc: true
...
+  // ...
+  release,
+  dist
+});
+```
+
+If you need to use `codePush.getUpdateMetadata`, you will have to wait to initialize the Sentry SDK until the Promise is resolved.
+<Alert level="warning" title="Warning"><markdown>
+
+This method is not recommended as any errors or crashes that happen before the Sentry SDK is initialized will not be caught.
```
Comment (MimiDumpling) [inline src/docs/platforms/react-native/codepush.mdx:23]:
```suggestion
This method is not recommended as any errors or crashes that happen before the initialization of the Sentry SDK will not be caught.  
```

File: src/docs/platforms/react-native/codepush.mdx [hunk 3]
```diff
@@ -28,3 +49,5 @@ $ sentry-cli react-native appcenter YourApp ios ./build/CodePush --dist YourBuil
 ```
 
 Exporting the `SENTRY_PROPERTIES` will tell sentry-cli to use the properties in your project. Alternatively, you can either pass it via parameters or a global settings file. To find more about this refer to [Working with Projects](/cli/configuration/#sentry-cli-working-with-projects).
+
+If you still have issues with CodePush sourcemaps, you can refer to [Source Maps for Other Platforms](/platforms/react-native/sourcemaps/) to manually bundle and upload sourcemaps to Sentry.
```
Comment (MimiDumpling) [inline src/docs/platforms/react-native/codepush.mdx:53]:
```suggestion
If you still have issues with CodePush source maps, you can refer to [Source Maps for Other Platforms](/platforms/react-native/sourcemaps/) to manually bundle and upload source maps to Sentry.
```

**Comment by marandaneto** [comment]
@jennmueng press the button if you feel its done :)

**Comment by jennmueng** [comment]
@marandaneto Yeah, I have this fear of breaking everything every time I merge 😅

**Pull request merged**